types/cytoscape/cytoscape-tests.ts

```diff
@@ -538,10 +538,18 @@ eles.difference(collNodes).abscomp().intersection(collSel).symdiff(collNodes);
 const diff = collSel.diff(collNodes);
 cy.collection().merge(diff.left).merge(diff.right).merge(diff.both).unmerge(collSel).filter((ele, i, eles) => true);
 
+nodes.map(n => n.degree(false));
+edges.map(e => e.source());
+eles.map(e => e.id());
+
 eles.sort((a, b) => 1).map((ele, i, eles) => [i, ele]);
 eles.reduce<any[]>((prev, ele, i, eles) => [...prev, [ele, i]], []).concat(['finish']);
 const min = eles.min((ele, i, eles) => ele.id.length + i); min.ele.scratch('min', min.value).scratch('min').value;
 const max = eles.max((ele, i, eles) => ele.id.length + i); max.ele.scratch('max', max.value);
+nodes.min(n => n.degree(false));
+nodes.max(n => n.degree(false));
+eles.max(n => n.source().id().length);
+eles.max(n => n.source().id().length);
```

Comment:
It doesn't seem to be right, because `eles` is a collection of any elements, both nodes and edges, and for nodes `source` is undefined. Seems that the type on `n` is wrongly inferred as `EdgeSingular`. Maybe just write `edges.min` and `edges.max` for now, just to have tests consistent, and later I'll look into it closer?